### DISCUSSION TITLE
[FR] Add multiple entries to a category at once

### DISCUSSION THREAD

**Discussion by sm9** [craftcms/cms#11315] [Ideas]
Clients have asked us a few times in the past year if it's possible to add multiple entries to a category at once, so I'm just adding this as a feature request.

Here's how I would imagine this working from a user's point of view:

1. On an an entries list view (e.g. admin/entries), you can already tick the checkboxes to select multiple entries.
2. After ticking multiple entries, click the Actions/Settings button at the top of the page, and a new option of 'Add to Category(ies)' would appear in that dropdown.
3. Clicking the 'Add to Category(ies)' option would then make a Category popup or slideout appear, and the user can then choose which category(ies) to add these entries to.

**Comment by thupsi** [comment]
Hi! There is a bulk editing feature in the roadmap, slated for 4.x., so that could cover your use case. Let's hope it lands soon. There is also a bulk edit plugin, but it's not ported to Craft 4 yet.

**Reply by sm9** [reply]
Hi, thanks for letting me know that a bulk editing feature is in the pipeline. That sounds like it will do the trick.

**Reply by brandonkelly** [reply]
Yep – you can follow along at #8515.
